package.json

```diff
@@ -46,10 +46,11 @@
     "generate-validate-bundle": "generate-validate-bundle -m '(?!^.+-factory\\.js$)(?=^.+\\.js$)' -d lib/validators -h resources/bundle-header.txt --recursive --filter lib/main.js"
   },
   "dependencies": {
-    "marc-record-js": "^0.1.1",
```

Comment:
There's a newer major version available for this module (_0.3.2_). The caret (`^`) only raises version up to the left-most non-zero digit and since the package release is under 1.0.0 the pattern won't really work. Maybe it's better to change the pattern until 1.0.0 is released for this module and then use the caret pattern again: `0.3.2 - 1.0.0`